controller/settingscontroller.php

```diff
@@ -107,38 +124,38 @@ public static function getSetting($settingName, $defaultValue = null,
      *              ]
      */
     public function getSettings($getFileContents = true, $isAdminForm = false) {
-        $data = array();
-        $adminSettings = array();
+        $data = [];
+        $adminSettings = [];
 
-        $appConfig = \OC::$server->getAppConfig();
-        $appId = Application::APP_ID;
+        $appConfig = $this->appConfig;
+        $appId = $this->appName;
 
         $txtFileProp = $appId . 'TxtFile';
-        $adminSettings[$txtFileProp] = array();
-        $adminSettings[$txtFileProp]['value'] = self::getSetting($txtFileProp,
```

Comment:
No need to use static methods. Avoid them at all costs, see https://r.je/static-methods-bad-practice.html